podium-gateway/src/main/webapp/app/shared/request/request-base.ts

```diff
@@ -15,17 +15,31 @@ import { User } from '../user/user.model';
 import { RequestDetail } from './request-detail';
 
 export class RequestBase implements Request {
-    public dateCreated?: Date;
-    public dateLastModified?: Date;
-
-    public uuid?: string;
-    public id?: string;
-    public status?: RequestStatus;
-    public requestDetail?: RequestDetail;
-    public organisations?: Organisation[];
-    public requester?: User;
-
-    constructor() {
+    dateCreated?: Date;
+    dateLastModified?: Date;
+    uuid?: string;
+    id?: string;
+    status?: RequestStatus;
+    requestDetail?: RequestDetail;
+    organisations?: Organisation[];
+    requester?: User;
 
+    constructor(dateCreated?: Date,
```

Comment:
See organisation constructor comment. Not necessary to build the constructor with args. Defaults can be set (if necessary) above where the variables are defined.